Dockerfiles/agent/DOCKERHUB.md

```diff
@@ -0,0 +1,61 @@
+## Supported architectures
+
+[`amd64`](https://hub.docker.com/r/datadog/agent-amd64) , [`arm64v8`](https://hub.docker.com/r/datadog/agent-arm64)
+
+## Supported versions
+
+#### Agent 7
+
+The Datadog agent, including a Python 3 interpreter for custom checks.
+
+Relevant tags are:
+
+- `7` , `7-jmx`
+- `7.X.X` , `7.X.X-jmx`
+- `latest` , `latest-jmx`
```

Comment:
```suggestion
- `latest` , `latest-jmx` --> use these for following the latest agent release, but keep in mind that it will update automatically to the next major release, which will have breaking changes. Major releases are very infrequent (less than one per year)
```